src/controller/python/chip/server/__init__.py

```diff
@@ -16,23 +16,23 @@
 
 import ctypes
 
-from chip import native
-from chip.native import PostAttributeChangeCallback, c_PostAttributeChangeCallback
+from .. import native as MatterNative
+from ..native import PostAttributeChangeCallback
```

Comment:
Why not simplify this further:

```
from ..native import PostAttributeChangeCallback, c_PostAttributeChangeCallback, Library
```

and remove MatterNative ?

Reply:
Thanks, all comments addressed.